fineract-provider/src/main/java/org/apache/fineract/interoperation/data/ExtensionData.java

```diff
@@ -23,13 +23,14 @@
 
 import com.google.gson.JsonObject;
 import java.util.Arrays;
+import java.util.List;
 import javax.validation.constraints.NotNull;
 import org.apache.fineract.infrastructure.core.data.DataValidatorBuilder;
 import org.apache.fineract.infrastructure.core.serialization.FromJsonHelper;
 
 public class ExtensionData {
 
-    public static final String[] PARAMS = { PARAM_KEY, PARAM_VALUE };
+    public static final List<String> PARAMS = List.copyOf(Arrays.asList(PARAM_KEY, PARAM_VALUE));
```

Comment:
I'm not convinced this `copyOf()` is actually really needed here, but it doesn't really matter that much (here in a `static final` constant), so fine for me... not a reason to hold back merging this, if that's the only feedback.